pliers/stimuli/base.py

```diff
@@ -208,9 +211,9 @@ def __str__(self):
 
     def to_df(self):
         def _append_row(rows, history):
-            rows.append(history[:-2])
-            if history[-1]:
-                _append_row(rows, history[-1])
+            rows.append(history[:-3])
```

Comment:
Since `history` is a namedtuple, we should probably index the attributes by name, which will improve clarity and minimize future edits

Reply:
Is there an easy way to do this when its slicing [:-3]?

Reply:
you could look up the attribute's index and then use that, but that seems like overkill. For slicing, I think leaving it as-is is fine.